src/app/Providers/components/ProvidersTable/CNV/CNVProvidersTable.tsx

```diff
@@ -13,11 +13,12 @@ import {
 import { DatabaseIcon } from '@patternfly/react-icons';
 import tableStyles from '@patternfly/react-styles/css/components/Table/table';
 import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
-import { useSortState, usePaginationState, useSelectionState } from '@app/common/hooks';
+import { useSortState, usePaginationState } from '@app/common/hooks';
 import CNVProviderActionsDropdown from './CNVProviderActionsDropdown';
 import { ICNVProvider } from '@app/Providers/types';
 import ProviderStatus from '../ProviderStatus';
 import './CNVProvidersTable.css';
+import { useSelectionState } from '@konveyor/lib-ui';
```

Comment:
It doesn't really matter, but out of personal preference I like to try and keep the npm imports above the local imports. We can leave it though, this isn't a hill I need to die on